core/server/proxy/src/main/java/alluxio/proxy/s3/S3Handler.java

```diff
@@ -89,6 +90,9 @@ public class S3Handler {
   private static final Logger LOG = LoggerFactory.getLogger(S3Handler.class);
   private static final ThreadLocal<byte[]> TLS_BYTES =
           ThreadLocal.withInitial(() -> new byte[8 * 1024]);
+  private static final int MAX_POSITION_READ_LENGTH = 4 * Constants.MB;
+  public static final int USE_POSITION_READ_SIZE = (int) Math.min(MAX_POSITION_READ_LENGTH,
```

Comment:
Can you describe that the max value is 4MB in the property description? And the`PROXY_S3_USE_POSITION_READ_RANGE_SIZE` will never be larger than 4MB